### PR TITLE
Tailwind setup

### PR DESCRIPTION
**Pull request by benjaminbiome**

### CODE REVIEW

File: pages/api/trees/plantTree.js [hunk 1]
```diff
@@ -30,6 +30,7 @@ export default function handler(req, res) {
 
   fetch("https://api.sandbox.digitalhumani.com/tree", requestOptions)
     .then((response) => {
+      console.log("Tree planted!");
```
Comment (trizoza) [inline pages/api/trees/plantTree.js:33]: left console.log

File: tailwind.config.js [hunk 2]
```diff
@@ -1,37 +1,52 @@
...
 
 module.exports = {
   mode: "jit",
   purge: ["./pages/**/*.{js,ts,jsx,tsx}", "./components/**/*.{js,ts,jsx,tsx}"],
   darkMode: false, // or 'media' or 'class'
   theme: {
     extend: {
+      fontFamily: {
+        heading: ['"Baloo 2"', ...defaultTheme.fontFamily.sans],
+        sans: ['"Helvetica"', ...defaultTheme.fontFamily.sans],
```
Comment (trizoza) [inline tailwind.config.js:12]: did you need to add that main.css - wouldn't just changes in this config file do the job?

Comment (benjaminbiome) [reply tailwind.config.js:12]: Yeh i did in order to change the content in the notion render.

File: pages/api/trees/plantTree.js (hunk 1 above)
Comment (benjaminbiome) [reply pages/api/trees/plantTree.js:33]: meh, leave it in, i quite like it when testing

**Review by peterbiome — APPROVED** [review]

**Pull request closed without merging**